### ISSUE TITLE
Unable to identify the accessibility id's for some ios native elements

### ISSUE THREAD

**Issue by fdPoornima** [appium/appium-desktop#1063]
## Appium or Appium Desktop?

Appium Desktop

- [ Yes] I have verified that my issue does not occur with Appium and should be investigated as an Appium Desktop issue

## The problem

I am writing automation tests for a hybrid native IOS application that my colleagues are developing. We are trying to maximise the use of accessibility id's to make script execution as fast as possible. The developers have added the accessibility id's for all the elements , in the same way by adding the 'accessibilityIdentifier' in xcode. However , when I inspect with the Appium Desktop, for certain elements, the accessibilityIdentifier value that is set is displayed as name , with no accessibility id. For some elements, the accessibility id is set to what the developers have set with accessibilityIdentifier  in their code. Why is their a mismatch?

## Environment

- I am running Appium Desktop version v 1.13.0_.
- I am on (pick one):
    - [ YES)) Mac
    - [ ] Windows
    - [ ] Linux

**Comment by surekhapentela** [comment]
I am also facing same issue. I have tried with 1.14.0 beta, able to find elements but due to that unable to install the app through automation code.

**Comment by surekhapentela** [comment]
When can i expect the proper build to identify all the elements ?

**Comment by KazuCocoa** [comment]
I observed cases a test app has no accessibility id properly even if I exactly set it on the app. Giving `accessibilityLabel` on the app worked at the time.
So, I'd say it depends on iOS (XCTest.framework), unfortunately.
